NextToMeAPI/NextToMe.Services/MessageDeleteService.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using NextToMe.Database;
+using NextToMe.Database.Entities;
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace NextToMe.Services
+{
+    public class MessageDeleteService : BackgroundService
+    {
+        private static readonly TimeSpan _delayTime = TimeSpan.FromMinutes(1);
+        private readonly IServiceScopeFactory _serviceScopeFactory;
+
+        public MessageDeleteService(IServiceScopeFactory serviceScopeFactory)
+        {
+            _serviceScopeFactory = serviceScopeFactory;
+        }
+        protected override async Task ExecuteAsync(CancellationToken stoppingToken)
+        {
+
+            while (!stoppingToken.IsCancellationRequested)
+            {
+                DeleteOldMessagesFromDb();
+                await Task.Delay(_delayTime, stoppingToken);
+            }
+        }
+
+        private async void DeleteOldMessagesFromDb()
```

Comment:
```suggestion
        private async Task DeleteOldMessagesFromDb()
```
писать async void опасно, например, нельзя будет перехватить созданные внутри метода исключения.
Async void было сделано только для асинхронных обработчиков событий.
Подробнее можно почитать тут, https://docs.microsoft.com/ru-ru/archive/msdn-magazine/2013/march/async-await-best-practices-in-asynchronous-programming